use-cases.md

```diff
@@ -153,8 +153,9 @@ func BusinessFunction1(ctx context.Context, arg1...) {
 }
 
 func BusinessFunction2(ctx context.Context, arg1...) {
-    parentSpan := SpanFromContext(ctx)
-    childSpan := opentracing.StartChildSpan(parentSpan, ...)
+    parentSpan := opentracing.SpanFromContext(ctx)
+    childSpan := opentracing.StartSpan(
+        "...", opentracing.BelongsTo(parentSpan.Context()), ...)
```

Comment:
`belongs_to` is interesting as it doesn't imply a hierarchy, and maybe I could put my grandparent there, because I indirectly do.. Not sure if this flexibility is what was intended or not.